src/Evolution/Systems/GrMhd/ValenciaDivClean/KastaunEtAlHydro.cpp

```diff
@@ -223,7 +223,7 @@ KastaunEtAlHydro<EnforcePhysicality>::apply(
           const EquationsOfState::EquationOfState<true, THERMODIM(data)>&    \
               equation_of_state);
 
-GENERATE_INSTANTIATIONS(INSTANTIATION, (1, 2), (true, false))
```

Comment:
This shouldn't compile for dim 3 because `specific_internal_energy_lower_bound` takes 2 arguments, right?